test/gc-dialects/Unit/lit.cfg.py

```diff
@@ -0,0 +1,50 @@
+# -*- Python -*-
```

Comment:
Maybe we don't need the multiple directories like `gc-dialects` to share the `lit` python files for other components like transforms/passes, runtime and others?

Reply:
The C++-based unittest is a special case. Filecheck tests don't need this lit config file.